### ISSUE TITLE
Mentioning a mentionable role in a message results in an error

### ISSUE THREAD

**Issue by gerritwalther** [Kraigie/nostrum#30]
Hi,

I've a problem sending mentionable roles in a message. E.g.  the message `<@&123456>` where 123456 is the id of a mentionable role, will result in the following error:

```
18:19:14.063 [error] GenServer #PID<0.240.0> terminating
** (BadMapError) expected a map, got: "123456"
    (elixir) lib/map.ex:424: Map.get("123456", "avatar", nil)
    (poison) lib/poison/decoder.ex:53: anonymous fn/3 in Poison.Decode.transform_struct/4
    (stdlib) lists.erl:1263: :lists.foldl/3
    (poison) lib/poison/decoder.ex:52: Poison.Decode.transform_struct/4
    (poison) lib/poison/decoder.ex:24: anonymous fn/5 in Poison.Decode.transform/4
    (elixir) lib/enum.ex:1826: Enum."-reduce/3-lists^foldl/2-0-"/3
    (poison) lib/poison/decoder.ex:24: Poison.Decode.transform/4
    (poison) lib/poison/decoder.ex:68: anonymous fn/6 in Poison.Decode.do_transform_struct/4
```

Here is a minimal project that reproduces the error: https://github.com/gerritwalther/nostrum_role_test.
It requires permission to create a role in the discord channel, but you can easily refactor it to use an existing role.

As far as I can see the problem occurs when handling the response from the Discord channel and Nostrum tries to decode the response with poison: https://github.com/kraigie/nostrum/blob/master/lib/nostrum/api.ex#L1755

Tell me if I can help to solve the problem, but my Elixir knowledge isn't that solid yet.

**Comment by Kraigie** [comment]
Fixed with 435b322159225b44ca1896dadb2f512cc6d34fe2, thanks!

**Comment by gerritwalther** [comment]
Thank you for the fix!
